src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -760,6 +762,52 @@ public void connect(String portName, int busIndex0, ModuleInst other, String oth
         }
     }
 
+    /**
+     * Connects a ModuleInst's single bit input port to GND or VCC
+     * 
+     * @param type     The static net type to connect the port to
+     * @param portName The name of the port
+     * 
+     */
+    public void connect(NetType type, String portName) {
+        connect(type, portName, -1);
+    }
+
+    /**
+     * Connects a ModuleInst's input port to GND or VCC
+     * 
+     * @param type     The static net type to connect the port to
+     * @param portName The name of the port
+     * @param busIndex The index of bit into the port, or -1 if it is a single bit
+     *                 port
+     */
+    public void connect(NetType type, String portName, int busIndex) {
+        if (type != NetType.GND && type != NetType.VCC) {
+            throw new RuntimeException("ERROR: Invalid NetType, should be GND or VCC, found: " + type);
+        }
+        Port input = getPort(busIndex == -1 ? portName : portName + "[" + busIndex + "]");
+        if (input == null) {
+            throw new RuntimeException(
+                    "ERROR: Couldn't find port " + portName + ", idx=" + busIndex + " on ModuleInst "
+                            + getName());
+        }
+        if (input.isOutPort()) {
+            throw new RuntimeException("ERROR: Port " + input.getName() + " on ModuleInst " + getName()
+                    + " is an output, cannot connect it to " + type);
+        }
+        Net physNet = type == NetType.GND ? getDesign().getGndNet() : getDesign().getVccNet();
+        EDIFNet logNet = EDIFTools.getStaticNet(type, getCellInst().getParentCell(), getDesign().getNetlist());
+        logNet.createPortInst(input.getName(), getCellInst());
```

Comment:
Do you need to handle the case where the port inst is already driven by a different logical net? (As you currently do below for the physical net)

Reply:
I've added code to handle this and updated the test accordingly.